daemon/config/config_solaris.go

```diff
@@ -31,17 +31,18 @@ type bridgeConfig struct {
 
 // InstallFlags adds command-line options to the top-level flag parser for
 // the current process.
-func (config *Config) InstallFlags(flags *pflag.FlagSet) {
+func (conf *Config) InstallFlags(flags *pflag.FlagSet) {
```

Comment:
Is this still called? It seems like it was removed from the other platforms, so I think this might be broken on solaris.